### PR TITLE
Use sensible defaults for environment variables

### PR DESCRIPTION
**Pull request by doismellburning**

### CODE REVIEW

File: lightbluetent/app.py [hunk 1]
```diff
@@ -3,7 +3,10 @@
 from . import admin, home
 
 
-def create_app(config_name):
+def create_app(config_name=None):
+
+    if config_name == None:
+        config_name = "Production"
```
Comment (edwinbalani) [inline lightbluetent/app.py:9]: Intentional capital P here?  I feel like all-lowercase is unwritten convention

Comment (edwinbalani) [reply lightbluetent/app.py:9]: Ah, I see now, this value gets `capitalize()`d later down anyway

Comment (doismellburning) [reply lightbluetent/app.py:9]: Semi-intentional - it's used for a class name later (which I'd ultimately like to remove) but I have no strong feelings - you're right to note that all-lowercase is fine and used elsewhere

Happy to change, but feels legit to leave as-is too

Comment (doismellburning) [reply lightbluetent/app.py:9]: Ah, thanks for merging regardless!

**Pull request merged**